packages/forms/src/Components/Concerns/HasState.php

```diff
@@ -186,6 +187,17 @@ public function getState()
         return $state;
     }
 
+    public function getOld()
+    {
+        $state = request('serverMemo.data.' . $this->getStatePath());
+
+        if (blank($state) || ! Livewire::isLivewireRequest()) {
```

Comment:
`Livewire::isLivewireRequest()` should probably be the first check before `$state = `